app/assets/javascripts/jquery.plug-google-content.js

```diff
@@ -38,7 +38,7 @@
         $.ajax({
           type: 'GET',
           url: batchBooksApiUrl,
-          async: false,
+          async: true,
```

Comment:
Not really, @jchristo4 added that change (although I'm not sure he's available to explain why).